tests/bulk_create/tests.py

```diff
@@ -35,22 +38,48 @@ def test_efficiency(self):
         with self.assertNumQueries(1):
             Country.objects.bulk_create(self.data)
 
-    def test_inheritance(self):
+    def assert_inheritance_error(self, *args, **kwargs):
+        """
+        Like assertRaisesMessage but with the right arguments for an inheritance
+        error already curried.
+        """
+        return self.assertRaisesMessage(
```

Comment:
Now that I look at this, I think I would combine the 3 cases that use this in 1 test method to avoid the helper method. We prefer using the context manager version of `assertRaisesMessage` for better readability.